### ISSUE TITLE
System.ObjectDisposedException: Não é possível acessar um objeto descartado.

### ISSUE THREAD

**Issue by wesleyrodrigues** [jonaskohl/CapsLockIndicator#54]
> Error when pressing caps lock.
```
System.ObjectDisposedException: Não é possível acessar um objeto descartado.
Nome do objeto: 'IndicatorOverlay'.
   em System.Windows.Forms.Control.CreateHandle()
   em System.Windows.Forms.Form.CreateHandle()
   em System.Windows.Forms.Control.get_Handle()
   em CapsLockIndicatorV3.IndicatorOverlay..ctor(String content, Int32 timeoutInMs, Color bgColour, Color fgColour, Color bdColour, Font font, IndicatorDisplayPosition position, Int32 indOpacity)
   em CapsLockIndicatorV3.MainForm.ShowOverlay(String message, Boolean isActive)
   em CapsLockIndicatorV3.MainForm.UpdateTimerTick(Object sender, EventArgs e)
   em System.Windows.Forms.Timer.OnTick(EventArgs e)
   em System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   em System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```
![image](https://user-images.githubusercontent.com/32652300/91639585-af891280-e9e5-11ea-802d-2904745f7a5e.png)

**Comment by jonaskohl** [comment]
This is a known issue, and I'm already working on a fix!

**Comment by jonaskohl** [comment]
Should be fixed in 3.9.0.0. If not, please reopen.
